help/en/html/doc/Technical/CI-status.shtml

```diff
@@ -106,10 +106,18 @@
 
         <tr>
             <td>
-                <a href="http://builds.jmri.org/jenkins/job/Development/job/Builds/">Builds</a> - Unit Test Results<br>
+                <a href="http://builds.jmri.org/jenkins/job/Development/job/Builds/">Builds</a> - Unit Tests Run and Skipped<br>
                 <a href="http://builds.jmri.org/jenkins/job/Development/job/Builds/lastBuild/testReport/">
                     <img src="http://builds.jmri.org/jenkins/job/Development/job/Builds/test/trend">
                 </a>
+http://builds.jmri.org/jenkins/job/Development/job/Builds/test/trend?failureOnly=true
```

Comment:
@bobjacobsen 
This line seems wrong. It's a link, but not included in a `<a>` tag or `<img>` tag.

And it looks wrong on the [CI status page](http://jmri.org/help/en/html/doc/Technical/CI-status.shtml)